worktree_commit_test.go

```diff
@@ -8,9 +8,15 @@ import (
 	"gopkg.in/src-d/go-git.v4/plumbing/storer"
 	"gopkg.in/src-d/go-git.v4/storage/memory"
 
+	"bytes"
 	. "gopkg.in/check.v1"
 	"gopkg.in/src-d/go-billy.v4/memfs"
+	"gopkg.in/src-d/go-billy.v4/osfs"
 	"gopkg.in/src-d/go-billy.v4/util"
+	"gopkg.in/src-d/go-git.v4/storage/filesystem"
+	"io/ioutil"
+	"os"
```

Comment:
Same here, but standard library imports in the first block, together with the others.
Same for go-git imports, put them in the same block as the other go-git imports.